api/services/itemService.js

```diff
@@ -1,13 +1,19 @@
 const { itemDao } = require("../models");
 
 const teabags = async (sorting, tasting_notes, price, category) => {
-    return await itemDao.getTeabags(sorting, tasting_notes, price, category);
-}
+  return await itemDao.getTeabags(sorting, tasting_notes, price, category);
```

Comment:
```suggestion
  return itemDao.getTeabags(sorting, tasting_notes, price, category);
```
수정님 코드가 아닌, 수현님 코드지만 await가 필요없는 코드이므로 대신 지워주세요~!